addon/services/csv.js

```diff
@@ -5,84 +5,82 @@ const defaultConfig = {
   fileName: 'export.csv',
   separator: ',',
   withSeparator: true
-}
+};
 
 export default Ember.Service.extend({
 
   export: function (data, options) {
-
     options = optionize(options, defaultConfig);
 
-    function JSON2CSV(objArray) {
-      var array = typeof objArray != 'object' ? JSON.parse(objArray) : objArray;
+    var csv = this.jsonToCsv(data, options);
 
-      var str = '';
-      var line = '';
+    saveAs(new Blob([csv],{type:"text/csv;charset=utf-8"}), options.fileName);
```

Comment:
I'm assuming this override is the same as https://github.com/Neuromobile/ember-cli-data-export/commit/fecec42e2d1739be079420632a764f7fc761604e?

Reply:
Yes, it is. For some reason, [SafeytyCulture/ember-](https://github.com/SafetyCulture/ember-cli-data-export) includes that commit.